superset/commands/database/importers/v1/utils.py

```diff
@@ -75,7 +79,11 @@ def import_database(
         ssh_tunnel = None
 
     # TODO (betodealmeida): we should use the `CreateDatabaseCommand` for imports
-    add_permissions(database, ssh_tunnel)
+
+    try:
+        add_permissions(database, ssh_tunnel)
+    except SupersetDBAPIConnectionError as ex:
```

Comment:
This might occur by other connection errors and not only the credentials one, right? So we would be affecting other engines too and not just BigQuery

Reply:
Could we instead use the [custom_errors in the db_engine_spec](https://github.com/apache/superset/blob/master/superset/db_engine_specs/bigquery.py#L183) and map the credentials one to something we could target individually?

Reply:
Correct, this would affect other DBs. While this bug emerged in the specific case of BigQuery, there are other DBs that may error as well when trying to set up catalogs on import. Regardless of the DB, the inability to add catalogs shouldn't necessarily import to fail since catalogs can be added later when updating the DB with valid creds.

Reply:
Ideally we'd only allow the exception if the database is configured for OAuth2. But even more ideally we'd do what the TODO says, and use `CreateDatabaseCommand` for the imports, since the command already handles edge cases like this. So I think this is OK for now.